packages/server/lib/gui/auth.ts

```diff
@@ -228,5 +228,6 @@ const start = (onMessage, utmCode, onLogin) => {
 
 export = {
   start,
+  stopServer,
```

Comment:
This will slightly change the behavior of the auth flow. By stopping the server, we're changing the value of `buildLoginRedirectUrl` when login is cancelled by changing the random port number that is returned and used as part of the OAuth redirect URL. 

I don't think it's a big deal, since it will only affect the unlikely use case where a user initiates the login flow, cancels it, restarts it, and then tries to login using the previous tab opened by the first `shell.openExternal` call. But I figured it's worth nothing.